apps/nextjs-website/src/components/HowTo.tsx

```diff
@@ -0,0 +1,35 @@
+'use client';
+import { HowToProps } from '@pagopa/pagopa-editorial-components/dist/components/HowTo';
+import { HowTo as HowToEC } from '@pagopa/pagopa-editorial-components';
+import * as MuiIcons from '@mui/icons-material';
+import { HowToSection } from '@/lib/fetch/types/PageSection';
+
+const makeHowToProps = ({
+  rowMaxSteps,
+  link,
+  steps,
+  ...rest
+}: HowToSection): HowToProps => ({
+  ...(rowMaxSteps && { rowMaxSteps }),
+  ...(link && {
+    link: {
+      href: link.href,
+      label: link.text ?? link.href,
+    },
+  }),
+  steps: steps.map((step) => ({
+    title: step.title,
+    description: step.description, // TODO: Parse rich text (markdown)
+    ...(step.icon && {
+      stepIcon: {
+        icon: step.icon as keyof typeof MuiIcons,
+        color: step.iconColor,
+      },
+    }),
+  })),
+  ...rest,
+});
+
+const HowTo = (props: HowToSection) => <HowToEC {...makeHowToProps(props)} />;
```

Comment:
I would probably add a `<section/>` surrounding `HowToEC` since it's needed for internal links (it's more inside the scope of this PR rather than the style one that will come next).